### PR TITLE
cleanup `client` package

### PR DESCRIPTION
**Pull request by dhrubabasu**

### CODE REVIEW

File: client/types.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ type infoPeersResponse struct {
 
 type ContractInfo struct {
 	Symbol   string `json:"symbol"`
-	Decimals uint8  `json:"decimals"`
+	Decimals int32  `json:"decimals"`
```
Comment (patrick-ogrady) [inline client/types.go:24]: Why not just do `uint` or `int` here? by default it is already 32.

Comment (dhrubabasu) [reply client/types.go]: removed it altogether and am just using `types.Currency`: 0d901a2

File: client/contract.go [hunk 2]
```diff
@@ -31,35 +32,37 @@ func NewContractClient(endpointURL string) (*ContractClient, error) {
...
 			symbol = UnknownERC20Symbol
 			decimals = UnknownERC20Decimals
 		} else {
 			symbol = UnknownERC721Symbol
 			decimals = UnknownERC721Decimals
 		}
 	}
-	contractInfo := &ContractInfo{Symbol: symbol, Decimals: decimals}
+
+	currency := &types.Currency{
```
Comment (patrick-ogrady) [inline client/contract.go:60]: I don't like the partial population of this. We should use the existing code (with a different var name)

Comment (dhrubabasu) [reply client/contract.go]: resolved in 1c42325

**Comment by dhrubabasu** [comment]
ready for review now

File: client/client.go [hunk 3]
```diff
@@ -60,11 +58,6 @@ func NewClient(endpoint string) (Client, error) {
 		return nil, err
 	}
 
-	evmlogs, err := NewEvmLogsClient(endpointURL)
-	if err != nil {
-		return nil, err
-	}
-
 	contract, err := NewContractClient(endpointURL)
```
Comment (patrick-ogrady) [inline client/client.go:61]: Is there a reason we can't pass the client in here?

Comment (dhrubabasu) [reply client/client.go]: fixed in 14bdc3d

**Review by patrick-ogrady — APPROVED** [review]

**Pull request merged**